src/app.js

```diff
@@ -9,7 +9,7 @@ import 'react-dates/initialize';
 import Helmet from 'react-helmet';
 import { BrowserRouter, StaticRouter } from 'react-router-dom';
 import { Provider } from 'react-redux';
-import mapValues from 'lodash/mapValues';
+import { difference, mapValues } from 'lodash';
```

Comment:
Don't export anything directly from Lodash. 
It is quite a big library and since we use a very small portion of functions it provides, we save almost 15 KB of gzipped javascript if we don't import it directly.